src/app/pages/checkout-review/checkout-review/checkout-review.component.ts

```diff
@@ -16,16 +17,18 @@ export class CheckoutReviewComponent implements OnInit {
   @Input() submitting: boolean;
   @Output() createOrder = new EventEmitter<void>();
 
-  form: FormGroup;
+  form = new FormGroup({});
+  model: { termsAndConditions: false };
```

Comment:
model = { termsAndConditions: false };
if you initiate the model correctly the required validator is not needed and the pattern validator does the job.